src/pages/clouddocs/temporal_overview.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Temporal Overview (Preview)"
+title: "Temporal Overview"
```

Comment:
For the left navigation content column, you will also need to update (e.g. lines 162, 164) [https://github.com/merative/acd-containers/blob/master/src/data/nav-items.yaml](https://github.com/merative/acd-containers/blob/master/src/data/nav-items.yaml)